Quake/common.c

```diff
@@ -2879,6 +2881,13 @@ unsigned COM_HashString (const char *str)
 	return hash;
 }
 
+static size_t mz_zip_file_read_func(void *opaque, mz_uint64 ofs, void *buf, size_t n)
+{
+	if (SDL_RWseek((SDL_RWops*)opaque, (Sint64)ofs, RW_SEEK_SET) < 0)
+		return 0;
+	return SDL_RWread((SDL_RWops*)opaque, buf, 1, n);
```

Comment:
Code is a bit inconsistent with other file IO. I've seen either just fopen or Sys_FileOpenRead

Reply:
SDL2 RWops works with 64 bit file sizes and offsets, so does miniz:
I didn't want to clobber miniz 64 bit values and wanted something
compatible with it.